### DISCUSSION TITLE
Issue with SMTP

### DISCUSSION THREAD

**Discussion by roshanjonah** [chatwoot/chatwoot#3327] [Self hosted] [answered]
I am using Helm Charts, I declared the following variables to use Postmark to deliver emails. 

```
 MAILER_SENDER_EMAIL: "notify@xxx.com"
 SMTP_ADDRESS: "smtp.postmarkapp.com"
 SMTP_AUTHENTICATION: "cram_md5"
 SMTP_USERNAME: "PM-T-outbound-xxx"
 SMTP_PASSWORD: "SbV3hSQDsgWv-xxx"
 SMTP_ENABLE_STARTTLS_AUTO: "true"
 SMTP_OPENSSL_VERIFY_MODE: "none"
 SMTP_PORT: "587"
 ```
 
 But for some reason, none of the emails would go out. When I try to reset the password link on the UI to trigger email sending, it gives me the following error in the 'worker' logs - https://justpaste.it/2k7e0

**Comment by vishnu-narayanan** [comment, marked as answer]
```cce2bc97-2af7-4002-86c2-7cf712dc9efa) from Sidekiq(mailers) in 33.58ms: Errno::ECONNREFUSED (Connection refused - connect(2) for "" port 587):```

From the error logs, it seems Sidekiq is having issues connecting to your SMTP service. Could you confirm the credentials and access? 

Ref: https://postmarkapp.com/support/article/1116-troubleshooting-common-connection-issues
